tests/verifyAuroraPostgreSQLRemoteInstance_test.js

```diff
@@ -0,0 +1,94 @@
+const assert = require('assert');
+const { NODE_TYPE, SERVICE_TYPE } = require('./helper/constants');
+
+const { remoteInstancesHelper, pmmInventoryPage } = inject();
+
+Feature('Monitoring Aurora PostgreSQL instances');
+
+const instances = ['postgres15aurora', 'postgres16aurora'];
+
+Before(async ({ I }) => {
+  await I.Authorize();
+});
+
+Data(instances).Scenario(
+  'Verify adding Aurora PostgreSQL RDS with specified Auto-discovery limit @aws @instances',
```

Comment:
Can you create coresponding tests in zephyr and add keys to your tests?

Reply:
updated

Reply:
Only for the new tests, for old we need to go trough all tests we have and recreate,